templates/base.html

```diff
@@ -206,10 +206,10 @@
                     </div>
                     <div class="content">
                         <!-- ko if: $root.getEagleIsReady() -->
-                            <!-- ko if: Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.ParameterTable -->
+                            <!-- ko if: Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.NodeParameterTable -->
                                 {% include 'parameter_table.html' %}
```

Comment:
There is a new file called `node_parameter_table.html` in this PR. But the code seems to still be including the old file `parameter_table.html`.